tapgarden/caretaker.go

```diff
@@ -612,18 +594,11 @@ func (b *BatchCaretaker) stateStep(currentState BatchState) (BatchState, error)
 			return 0, fmt.Errorf("unable to deserialize genesis "+
 				"PSBT: %w", err)
 		}
-		changeOutputIndex := b.cfg.Batch.GenesisPacket.ChangeOutputIndex
 
-		// If the change output is first, then our commitment is second,
-		// and vice versa.
-		// TODO(jhb): return the anchor index instead of change? or both
-		// so this works for N outputs
-		b.anchorOutputIndex, err = extractAnchorOutputIndex(
-			b.cfg.Batch.GenesisPacket,
-		)
-		if err != nil {
-			return 0, err
-		}
+		// Unpack output indexes.
+		changeOutputIndex := b.cfg.Batch.GenesisPacket.ChangeOutputIndex
```

Comment:
nit: move `genesisPacket := b.cfg.Batch.GenesisPacket` up here to shorten these?